test/helpers/MarketHandler.sol

```diff
@@ -0,0 +1,227 @@
+/* solhint-disable contract-name-camelcase, func-name-mixedcase, not-rely-on-time */
```

Comment:
this contract is what will be called and fuzzed by the fuzzing engine (its external functions at least).  randomized call sequences will be made with equal probability among external functions up to the number of calls in the `depth` setting in config. (currently this handler only has 1 external function and my depth was set to 1 so each run will simply call `endToEnd` once, but I'm going to change that)

"handler" contracts like this are used to customize and arrange logic in a more complex system so that fuzzing inputs can be meaningful and not just a bunch of meaningless reverts.  sky's the limit in here.